assets/src/component/sw-braintree-app-settings/sw-braintree-app-settings-general.vue

```diff
@@ -7,6 +7,7 @@
     <div class='sw-braintree-app-settings-general__threeDSecuredEnforced'>
         <mt-switch
             class='sw-braintree-app-settings-general__threeDSecureEnforced__input'
+            :bordered='true'
```

Comment:
```suggestion
            :bordered
```
It is the more idiomatic and concise